runbot/models/build.py

```diff
@@ -736,7 +736,7 @@ def _docker_run(self, **kwargs):
             self._log('Preparing', 'Using Dockerfile Tag %s' % kwargs['image_tag'])
         containers_memory_limit = self.env['ir.config_parameter'].sudo().get_param('runbot.runbot_containers_memory', 0)
         if containers_memory_limit and 'memory' not in kwargs:
-            kwargs['memory'] = containers_memory_limit * 1024 ** 3
+            kwargs['memory'] = float(containers_memory_limit) * 1024 ** 3
```

Comment:
My last suggestion was mainly to add a cast to int afterward, I add no problem with the cast to float on the previous lines ;)
```suggestion
            kwargs['memory'] = int(float(containers_memory_limit) * 1024 ** 3)
            
```
It's working now but this would be safer in case docker makes change on the command line. Except that LGTM